core/src/main/java/com/scalar/db/storage/multistorage/MultiStorageAdmin.java

```diff
@@ -200,6 +207,48 @@ public void importTable(String namespace, String table) throws ExecutionExceptio
     getAdmin(namespace, table).importTable(namespace, table);
   }
 
+  @Override
+  public Set<String> getNamespaceNames() throws ExecutionException {
+    // Only return existing namespaces that are listed in the namespace mapping configuration or
+    // when they belong to the default storage
+    //
+    // For example, if the storages contain the following namespaces :
+    // - mysql : mysqlStorageAdmin.getNamespaceNames() = [ns1, ns2]
+    // - cassandra : cassandraStorageAdmin.getNamespaceNames() = [ns3]
+    // - cosmos : cosmosStorageAdmin.getNamespaceNames() = [ns4, ns5]
+    // And the default storage is cassandra :
```

Comment:
```suggestion
    // And the default storage is cosmos :
```

Reply:
Thank you, I revised it in https://github.com/scalar-labs/scalardb/pull/989#discussion_r1286624632